### DISCUSSION TITLE
[Showcase] CSSUI

### DISCUSSION THREAD

**Discussion by zetareticoli** [gethyas/hyas#491] [Showcase]
### Site name

CSSUI

### Public URL

https://cssui.dev

### Short description

A pure CSS library for interactive components.

### Terms

- [X] I agree to the [Terms of Service](https://thulite.io/terms/).

**Comment by h-enk** [comment]
Thanks for sharing – super nice components!

**Comment by zetareticoli** [comment]
Thanks @h-enk 🙌
